packages/chain-adapters/src/ethereum/EthereumChainAdapter.test.ts

```diff
@@ -88,6 +89,25 @@ describe('EthereumChainAdapter', () => {
       }
     }
   })
+
+  describe('constructor', () => {
+    it('should return chainAdapter with no chainId', () => {
+      const adapter = new ethereum.ChainAdapter(args)
+      const chainId = adapter.getChainId()
+      expect(chainId).toEqual(VALID_CHAIN_ID)
+    })
+    it('should return chainAdapter with valid chainId', () => {
+      args.chainId = 'eip155:3'
+      const adapter = new ethereum.ChainAdapter(args)
+      const chainId = adapter.getChainId()
+      expect(chainId).toEqual('eip155:3')
+    })
+    it('should return chainAdapter with invalid chainId', () => {
```

Comment:
This isn't actually testing `this.chainId`, just that we (re)throw.
Please modify the name of the test:
```suggestion
    it('should throw if called with invalid chainId', () => {
```